server/routes/routerUtils.ts

```diff
@@ -1,20 +1,13 @@
 import { RequestHandler, Router } from 'express'
-import asyncMiddleware from '../middleware/asyncMiddleware'
 
 const getRequest =
```

Comment:
these `getRequest`/`postRequest` helpers now don’t do anything other than hide the true interface of the router methods.

i really wanna get rid of them if there’s appetite…?